optuna/integration/keras.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
+
+from typing import Dict  # NOQA
+
+import optuna
+
+try:
+    from keras.callbacks import Callback
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # KerasPruningExtension is disabled because Keras is not available.
+    _available = False
+    # This alias is required to avoid ImportError at KerasPruningExtension definition.
+    Callback = object
+
+
+class KerasPruningCallback(Callback):
+    """Keras callback to prune unpromising trials.
+
+    Example:
+
+        Add a pruning callback which observes validation losses.
+
+        .. code::
+
+            model.fit(X, y, callbacks=KerasPruningCallback(trial, 'val/loss'))
```

Comment:
I think `val/loss` is a typo for `val_loss` as we discussed in the previous review.

Reply:
Oops, It is a typo. I will fix it ASAP.